pennylane/_device.py

```diff
@@ -62,12 +62,20 @@ def __init__(self, wires=1, shots=1000):
 
     def __repr__(self):
         """String representation."""
-        return "{}.\nInstance: ".format(self.__module__, self.__class__.__name__, self.name)
+        return "<{} ({}, wires={}, shots={}) at {}>".format(
+            self.__class__.__name__, self.short_name, self.num_wires, self.shots, hex(id(self))
```

Comment:
I think its unnecessary to have both name and short name in the less verbose representation.
e.g., this prints the following:
```
<DefaultQubit (default.qubit, wires=1, shots=1000) at 0x7fa9e2e3b590>
```

Reply:
Agree!